applications/metis_application/custom_processes/metis_scalar_reorder_process.h

```diff
@@ -11,18 +11,12 @@
 #define  KRATOS_METIS_SCALAR_REORDER_PROCESS_H_INCLUDED
 
 
-
 // System includes
-#include <string>
-#include <iostream>
-
 
 // External includes
-
+#include <parmetis.h>
```

Comment:
i see that it is not you changing this...however the dependency should be on metis.h NOT on parmetis.h

Reply:
I will try to do this in a separate PR

Reply:
Hm this seems not to be trivial, apparently for metis-version < 5 parmetis is included
I don't know how/why this is done so I would prefer if someone else does to not break anything